RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -0,0 +1,63 @@
+# This is meant for items that have a date of 2023
+
+boolean auto_haveRockGarden()
+{
+	static item rockGarden = $item[Packet Of Rock Seeds];
```

Comment:
What's the benefit of using `static` here?

Reply:
Don't know. I saw it in a different garden and thought maybe gardens need to be handled differently than other campground items. I'm not sure that they do, but I don't have any other gardens to check out what they return in get_campground(). It looks like other ones might return the item you can get from the garden but not the base item, based on the code for auto_get_campground(). I don't think it hurts to have it as a static though?

Reply:
No doesn't hurt anything that I'm aware of, was just curious.